src/test/common/page/page.js

```diff
@@ -124,6 +124,15 @@ class Page {
     }
   }
 
+  async findByLinkText (linkText) {
```

Comment:
See my comment below about adding an id to the link, we then don't need this method and can use `findById` which will be less fragile should the text on the link change.

Reply:
Removed method.